bool is_true        # acceptance information (True/False)
uint32 sequence_id  # Received sequence ID
time received_time  # Received timestamp
string frame_id     # Received frame ID

# FireTarget message fields
float64 x           # x-coordinate
float64 y           # y-coordinate
float64 depth       # depth
uint8 status       # fire status
